test/games/strategy/triplea/delegate/StratBombTest.java

```diff
@@ -0,0 +1,127 @@
+import java.util.Collections;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import games.strategy.engine.data.Change;
+import games.strategy.engine.data.changefactory.ChangeFactory;
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.ITestDelegateBridge;
+import games.strategy.engine.data.PlayerID;
+import games.strategy.engine.data.Route;
+import games.strategy.engine.data.RouteScripted;
+import games.strategy.engine.data.Territory;
+import games.strategy.engine.data.Unit;
+import games.strategy.engine.data.UnitType;
+import games.strategy.engine.random.IRandomSource;
+import games.strategy.engine.random.ScriptedRandomSource;
+import games.strategy.test.TestUtil;
+import games.strategy.triplea.Constants;
+import games.strategy.triplea.TripleAUnit;
+import games.strategy.triplea.attachments.UnitAttachment;
+import games.strategy.triplea.player.ITripleAPlayer;
+import games.strategy.triplea.xml.LoadGameUtil;
+
+public class StratBombTest {
+  private GameData m_data;
+
+  @Before
+  public void setUp() throws Exception {
+    m_data = LoadGameUtil.loadTestGame(LoadGameUtil.TestMapXml.GLOBAL1940);
+  }
+
+  private ITestDelegateBridge getDelegateBridge(final PlayerID player) {
+    return GameDataTestUtil.getDelegateBridge(player, m_data);
+  }
+
+
+  @Test
+  public void TestBombingRaid2targets() {
+    final Territory germany = m_data.getMap().getTerritory("Western Germany");
+    final Territory uk = m_data.getMap().getTerritory("United Kingdom");
+    final PlayerID germans = GameDataTestUtil.germans(m_data);
+    final PlayerID british = GameDataTestUtil.british(m_data);
+    // add a unit
+    final Unit stratBomber = GameDataTestUtil.bomber(m_data).create(british);
+    final Unit tacBomber1 = GameDataTestUtil.tacBomber(m_data).create(british);
+    final Unit tacBomber2 = GameDataTestUtil.tacBomber(m_data).create(british);
+    m_data.performChange(ChangeFactory.addUnits(uk, Collections.singleton(stratBomber)));
+    m_data.performChange(ChangeFactory.addUnits(uk, Collections.singleton(tacBomber1)));
+    m_data.performChange(ChangeFactory.addUnits(uk, Collections.singleton(tacBomber2)));
+    final BattleTracker tracker = new BattleTracker();
+    List<Unit> attackers = new ArrayList<>();
+    attackers.add(stratBomber);
+    attackers.add(tacBomber1);
+    attackers.add(tacBomber2);
+    HashMap<Unit, HashSet<Unit>> targets = new HashMap<>();
+    TripleAUnit airfield = null;
+    TripleAUnit harbour = null;
+    TripleAUnit factory = null;
+
+    // Find facilities in territory
+    for( final Unit target : germany.getUnits().getUnits() ) {
+      switch( target.getType().getName() ) {
+        case "airfield"      : airfield = (TripleAUnit) target; 
+          break;
+        case "harbour"       : harbour = (TripleAUnit) target;
```

Comment:
If the AA fire is broken up between harbour and factory, what does throwing in airfield show? IE: in what cases would we see a test failure only because we included airfield in the test data?